doc/classes/AudioStreamPlayer3D.xml

```diff
@@ -66,8 +66,8 @@
 			The bus on which this audio is playing.
 		</member>
 		<member name="doppler_tracking" type="int" setter="set_doppler_tracking" getter="get_doppler_tracking" enum="AudioStreamPlayer3D.DopplerTracking" default="0">
-			Decides in which step the Doppler effect should be calculated.
-		</member>
```

Comment:
Documentation bug. (Don't you have `pre-commit-makerst` set up?)

Reply:
> Documentation bug. (Don't you have pre-commit-makerst set up?)

I do, but it looks like I made a mistake when editing the XML and it wasn't caught by makerst.